sympy/tensor/index_methods.py

```diff
@@ -12,6 +12,18 @@
 
 from __future__ import print_function, division
 
+ """
+
+ __future__ is a real module, and serves three purposes:
+
+To avoid confusing existing tools that analyze import statements and expect to find the modules they�re importing.
+To ensure that future statements run under releases prior to 2.1 at least yield runtime exceptions (the import of __future__ will fail, because there was no module of that name prior to 2.1).
```

Comment:
This line and the next line easily extends the 79-character limit.
Please be careful not to have too long lines. It doesn't follow the Python and SymPy styling guide
[PEP-8](https://www.python.org/dev/peps/pep-0008/)

Reply:
ok next time i will considered it